src/sec_edgar_bulker/core/exceptions.py

```
@@ -0,0 +1,47 @@
+
+class ConfigError(Exception):
+    """Raised when there is an error with the configuration."""
+    pass
+
+class ParsingError(Exception):
+    """Raised when there is an error parsing a file."""
+    pass
+
+class DownloadModeError(Exception):
+    """Raised when there is an error with the download mode."""
+    def __init__(self, message, details=None):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)